al2023neu.pkr.hcl

```diff
@@ -1,5 +1,14 @@
 locals {
   ami_name_al2023neu = "${var.ami_name_prefix_al2023}-neuron-hvm-2023.0.${var.ami_version_al2023}${var.kernel_version_al2023}-x86_64"
+  tags = {
```

Comment:
Should this line read as `default_tags = {` (like most of the other AMI recipe files), or is this intentional?